lib/widgets-page.php

```diff
@@ -58,6 +58,12 @@ function gutenberg_widgets_init( $hook ) {
 		)
 	);
 
+	wp_add_inline_script(
+		'wp-blocks',
+		sprintf( 'wp.blocks.setCategories( %s );', wp_json_encode( get_block_categories( 'widgets_editor' ) ) ),
```

Comment:
It should be `gutenberg_get_block_categories` for now.

Reply:
Oh whoops, I only changed it in the other file. Will push up another PR.

Reply:
https://github.com/WordPress/gutenberg/pull/31486 fixes. Thanks for noticing!